src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx

```diff
@@ -27,24 +23,48 @@ With Azion, you can customize the time the cache remains in the edge through *ti
+Azion provides a layer to cache content in the edge network, which stands between origin servers and the end user. When content is successfully cached at the edge, it receives a [cache status HIT](#status-definitions) and it can be directly delivered to your users from the edge node nearest to them, reducing the frequency of access to origin servers.
+
+In the event the requested content isn't available in cache, the Edge Caching module minimizes the impact by maintaining a keep-alive connection with the origin server whenever possible. This approach avoids the overhead of frequent TCP/IP handshakes. Regardless of the number of simultaneous requests made to Azion's edge nodes, each edge node only needs to request content from the origin once per MISS or other statuses.
 
-You can obtain the cache key of an application's resource through a request for the content URI and add the Azion debug header `Pragma: azion-debug-cache`:
+You can obtain the cache status of an edge application's resource by requesting the content URI and adding the Azion debug header `Pragma: azion-debug-cache`. A successful response will return an `X-Cache` header with the cache status, the IP of the edge node, and the protocol used in the request:
 
 ```bash
-curl -I https://yourdomain.com/resource-path/image.jpeg -H "Pragma: azion-debug-cache"
+X-Cache: HIT from 200.000.000.00 with HTTP/2.0
 ```
 
-A successful response will return an `X-Cache-Key` header with the cache key value, which based on the example above, would be:
+:::note
+Find out more about how to automate your browser to obtain other cache data in the [How to verify application cache indicators using Modheader for Google Chrome](/en/documentation/products/guides/check-page-cache-time/) guide.
+:::
 
-```
+### Status definitions
+
+| Status | Definition |
+| --- | --- |
+| **HIT** | Valid and up-to-date content is supplied directly from the edge cache. |
+| **MISS** | If the requested content isn't found in the cache at the edge, it's fetched from the origin. Through the response, the content might be cached for future requests. |
+| **EXPIRED** | The cached content has expired the TTL set at the edge. When the origin responds, the cached content is updated for future requests. |
+| **STALE** | When choosing to serve [stale cache](#stale-cache), if the origin fails to respond and the edge cache has expired, a stale version of the content is served. |
+| **UPDATING** | The cached content has expired and stale cache is being served, but the content is being updated in the origin. |
+| **BYPASS** | The edge requests the content from the origin directly instead of using a cached version due to the active [Bypass Cache](/en/documentation/products/edge-application/rules-engine/#bypass-cache) behavior. |
```

Comment:
```suggestion
| **HIT** | Valid and up-to-date content is supplied directly from the edge cache |
| **MISS** | If the requested content isn't found in the cache at the edge, it's fetched from the origin. Through the response, the content might be cached for future requests |
| **EXPIRED** | The cached content has expired the TTL set at the edge. When the origin responds, the cached content is updated for future requests |
| **STALE** | When choosing to serve [stale cache](#stale-cache), if the origin fails to respond and the edge cache has expired, a stale version of the content is served |
| **UPDATING** | The cached content has expired and a stale cache is being served, but the content is being updated in the origin |
| **BYPASS** | The edge requests the content from the origin directly instead of using a cached version due to the active [Bypass Cache](/en/documentation/products/edge-application/rules-engine/#bypass-cache) behavior |
```